build_tools/services.rb

```diff
@@ -10,10 +10,10 @@ class ServiceEnumerator
     MANIFEST_PATH = File.expand_path('../../services.json', __FILE__)
 
     # Minimum `aws-sdk-core` version for new gem builds
-    MINIMUM_CORE_VERSION = "3.205.0"
+    MINIMUM_CORE_VERSION = "3.206.0"
```

Comment:
I don't think we need to bump min core for this? If you rely on an old version of core, everything will continue to work as is.